src/api.c

```diff
@@ -162,6 +167,15 @@ API int seccomp_api_set(unsigned int level)
 		sys_set_seccomp_action(SCMP_ACT_LOG, true);
 		sys_set_seccomp_action(SCMP_ACT_KILL_PROCESS, true);
 		sys_set_seccomp_flag(SECCOMP_FILTER_FLAG_SPEC_ALLOW, true);
+	case 5:
```

Comment:
If the user notification and SPEC_ALLOW functionality was merged at the same time into Linux v5.0, it seems like we can leverage API level "4" for both.

Reply:
They weren't; SPEC_ALLOW went in as 4.17, so I'll leave it separate.